packages/ui-library/src/theme/variants/button-variants.ts

```diff
@@ -22,6 +22,7 @@ export const buildButtonVariant = (mode: 'light' | 'dark'): Components => ({
       },
       sizeSmall: {
         padding: `${SPACING(1.5)} ${SPACING(3)}`,
+        minWidth: SPACING(8),
```

Comment:
hold up cowboy, doesnt this break other stuff?

Reply:
The small should be as small as it can be

Reply:
You're right! This was just migrated from feat/earn, but 0 as min-width works just fine